src/DatePicker/CalendarToolbar.js

```diff
@@ -72,13 +76,17 @@ class CalendarToolbar extends Component {
       year: 'numeric',
     }).format(displayDate);
 
+    const nextButtonIcon = this.context.muiTheme.isRtl ? <NavigationChevronLeft /> : <NavigationChevronRight />;
+    const prevButtonIcon = this.context.muiTheme.isRtl ? <NavigationChevronRight /> : <NavigationChevronLeft />;
+
+
     return (
       <div style={styles.root}>
```

Comment:
That's going to make theme invariant from the direction logic. It's a simpler fix.